client/state/jitm/selectors.js

```diff
@@ -14,7 +14,7 @@ import { getSelectedSiteId } from 'state/ui/selectors';
  * @returns {Array} An array of jitms
  */
 export const getJITM = ( state, messagePath ) =>
-	get( state, [ 'jitm', 'sitePathJITM', messagePath + getSelectedSiteId( state ) ], [] );
+	get( state, [ 'jitm', 'sitePathJITM', messagePath + getSelectedSiteId( state ) ], null );
```

Comment:
Default state of `null` is a better indicator of "unfulfilled" state for a given `messagePath`. The `[]` state can be then understood to mean "Fulfilled: we queried the network for JITMs and there were none for this route.".